File: CompanyTalk_WebApi_v2/CompanyTalk_WebApi_v2/Models/LogEntrada.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CompanyTalk_WebApi_v2.Models
{
    public class LogEntrada
    {
        public LogEntrada(int id, int pessoaId, string dataLog, string horaLog)
        {
            this.Id = id;
            this.PessoaId = pessoaId;
            this.DataLog = dataLog;
            this.HoraLog = horaLog;
        }
        public int Id { get; set; }
        public int PessoaId { get; set; }
        public string DataLog { get; set; }
        public string HoraLog { get; set; }
        public Pessoa Pessoa { get; set; }
    }
}

File: CompanyTalk/src/app/models/LogEntrada.ts
import { Pessoa } from "./Pessoa";

export class LogEntrada {
  id!: number;
  pessoaId!: number;
  dataLog!: string;
  horaLog!: string;
  pessoa!: Pessoa;

  constructor(pessoa: Pessoa, dataLog: string, horaLog: string){
    this.pessoaId = pessoa.id;
    this.dataLog = dataLog;
    this.horaLog = horaLog;
  }
}
